RecoTracker/TkNavigation/test/NavigationSchoolAnalyzer_cfg.py

```diff
@@ -7,7 +7,8 @@
 # process.load("Configuration.StandardSequences.Geometry_cff")
 #process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
 #process.load('Configuration.Geometry.GeometryExtended2023D1Reco_cff')
-process.load('Configuration.Geometry.GeometryExtended2023D4Reco_cff')
+#process.load('Configuration.Geometry.GeometryExtended2023D4Reco_cff')
```

Comment:
@parbol strictly speaking we have no more 2023 geometries, so these lines are useless (even if harmless)

Reply:
Hi Fabio,

I have remove the reference to the 2023 geometries in a later commit.